randovania/data/json_data/prime1.txt

```diff
@@ -4082,11 +4253,17 @@ Asset id: 2263559682
 
 > Behind PB wall; Heals? False
   > Dock to Life Grove Tunnel
-      Power Beam and X-Ray Visor and Space Jump Boots
+      All of the following:
+          Power Beam
+          Any of the following:
+              Space Jump Boots
+              Morph Ball Bomb and Morph Ball and Combat/Scan Dash (Intermediate) and Bomb Jump (Advanced)
   > Pickup (X-Ray Visor)
-      Power Bomb and X-Ray Visor and Morph Ball
+      Power Bomb and Morph Ball
   > Pickup (Artifact of Chozo)
-      Morph Ball Bomb and Morph Ball and Boost Ball
+      All of the following:
+          Morph Ball and Boost Ball
+          Morph Ball Bomb or Power Bomb
```

Comment:
Opening with PBs should be knowledge beginner

Reply:
Should be fixed now